Source/Scene/GlobeDepth.js

```diff
@@ -0,0 +1,235 @@
+/*global define*/
```

Comment:
I get 91% coverage. The only code paths not executed are when depth textures are not supported or creating the framebuffer fails.